ci/quickstart-bitbucket-master-v5.json

```diff
@@ -5,11 +5,11 @@
     },
```

Comment:
What's the difference between this parameter set and `quickstart-bitbucket-default.json`?

Reply:
This one is used with master template with VPC (`quickstart-bitbucket-dc-with-vpc.template.yaml`)